### PR TITLE
Remove unused promises

### PR DESCRIPTION
**Pull request by Stojdza**
For easier maintenance remove unused functions which return promises.

### CODE REVIEW

File: lib/email-client.js [hunk 1]
```diff
@@ -12,30 +12,6 @@ export default class EmailClient {
 		this.mailbox = new Mailosaur.Mailbox( mailboxId );
 	}
 
-	deleteAllEmailByID( emailID ) {
```
Comment (brbrr) [inline lib/email-client.js]: This method is used here https://github.com/Automattic/wp-e2e-tests/blob/master/specs/wp-log-in-out-spec.js#L248

File: lib/pages/devdocs-page.js [hunk 2]
```diff
@@ -109,79 +86,4 @@ export default class DevdocsPage extends BaseContainer {
 			'document.querySelector( ".environment-badge" ).style.visibility = "hidden";'
 		);
 	}
-
-	screenshotAllElements( eyes ) {
```
Comment (brbrr) [inline lib/pages/devdocs-page.js]: This method is used here: https://github.com/Automattic/wp-e2e-tests/blob/master/specs-visdiff/wp-devdocs-visdiff.js.removed#L46

@alisterscott do know why do we keeping these `.removed` viss diff tests?

File: lib/push-client.js [hunk 3]
```diff
@@ -1,38 +1,6 @@
...
 import { listen } from 'push-receiver';
-import request from 'request-promise';
-
-/**
- * Marks a push token as approved on WPCOM API
- * @param {String} pushToken push token that recieved via a push notification on an app
- * @param {String} bearerToken user bearer token to be used with the API
- * @returns {Promise} a request result promise
- */
-export const approvePushToken = ( pushToken, bearerToken ) =>
```
Comment (brbrr) [inline lib/push-client.js]: This method is used here: https://github.com/Automattic/wp-e2e-tests/blob/master/specs/wp-log-in-out-spec.js#L167

File: lib/email-client.js (hunk 1 above)
Comment (Stojdza) [reply lib/email-client.js]: Also here:
https://github.com/Automattic/wp-e2e-tests/blob/master/specs/wp-log-in-out-spec.js#L323
https://github.com/Automattic/wp-e2e-tests/blob/master/specs/wp-log-in-out-spec.js#L390

File: lib/pages/devdocs-page.js (hunk 2 above)
Comment (Stojdza) [reply lib/pages/devdocs-page.js]: Also here: https://github.com/Automattic/wp-e2e-tests/blob/master/specs-visdiff/wp-devdocs-visdiff.js.removed#L63

File: lib/email-client.js (hunk 1 above)
Comment (Stojdza) [reply lib/email-client.js]: Also here:
https://github.com/Automattic/wp-e2e-tests/blob/master/specs/wp-log-in-out-spec.js#L323
https://github.com/Automattic/wp-e2e-tests/blob/master/specs/wp-log-in-out-spec.js#L390

**Comment by Stojdza** [comment]
I put back functions which are used in temporary disabled tests and disable them. 

@alisterscott What about `.removed` files? Are those permanently deprecated? If yes, I think that functions used there can be safely removed.

**Comment by alisterscott** [comment]
> What about .removed files? Are those permanently deprecated? If yes, I think that functions used there can be safely removed.

Yes, let's remove those files - we can add them back if we need to (and do promise stuff then)

**Pull request merged**